src/Altinn.Platform/Altinn.Platform.Authorization/Altinn.Authorization.ABAC/Utils/XacmlSerializer.cs

```diff
@@ -36,6 +37,60 @@ public static void WriteContextResponse(XmlWriter writer, XacmlContextResponse x
             writer.WriteEndElement();
         }
 
+        /// <summary>
+        /// Method to serialize a XACML Policy.
+        /// </summary>
+        /// <param name="writer">XML Writer</param>
+        /// <param name="xacmlPolicy">The XACML Policy to be serialized</param>
+        public static void WritePolicy(XmlWriter writer, XacmlPolicy xacmlPolicy)
```

Comment:
Could this have been an extension method for XacmlPolicy?

Reply:
I guess it could be implemented as a extension method yes. XacmlPolicy.GetXmlWriter() or XacmlPolicy.GetAsDataStream()?
which we then can pass to the blob storage repository implementation.
Didn't really think of that as a possible solution since we already had the XacmlSerializer implementation.

@elsand @TheTechArch Thoughts?

Reply:
It should still take the XmlWriter as input.

Reply:
It's fine that the method takes the writing target as input.

Reply:
Good point. It's been decided though that we'll merge as is for now at least. But refactoring to an extension method is something we can definitely consider